src/main/java/org/traccar/notificators/NotificatorFirebase.java

```diff
@@ -77,15 +83,15 @@ public void sendSync(long userId, Event event, Position position) {
             Context.getClient().target(url).request()
                     .header("Authorization", "key=" + key)
                     .async().post(Entity.json(message), new InvocationCallback<Object>() {
-                @Override
-                public void completed(Object o) {
-                }
-
-                @Override
-                public void failed(Throwable throwable) {
-                    LOGGER.warn("Firebase notification error", throwable);
-                }
-            });
+                        @Override
```

Comment:
Looks like formatting is not reverted here.

Reply:
Okat, I will try to merge it from master.

Reply:
Why do you need to merge it? Just revert this change.

Reply:
As it is automatically formatted by IDE so the overall file is affected.

Reply:
Theser are reverted in 7d5a18e.